src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/SponsorBlockBytecodePatch.kt

```diff
@@ -34,7 +34,7 @@ import com.android.tools.smali.dexlib2.iface.reference.StringReference
 
 @Patch(
     name = "SponsorBlock",
-    description = "Integrates SponsorBlock which allows skipping video segments such as sponsored content.",
+    description = "Integrates SponsorBlock, which can skip undesired video segments such as sponsored content.",
```

Comment:
```suggestion
    description = "Integrates SponsorBlock, which can skip over sponsors, intros, outros, subscription reminders, and other annoying parts of YouTube videos.",
```

Reply:
I feel like "Subscription reminder" doesn't properly relay the type of reminder, since nobody calls it a subscription on the YT platform

Perhaps `Subscribe reminder`?

Reply:
This description is a copy of SponsorBlock's description.